test/unit/helpers/bignum.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var bignum = require('../../../helpers/bignum');
+
+describe('bignum', function () {
+
+	var SEED = '782910138827292261791972728324982';
+	var BUFFER_SEED = Buffer.from('Jpm3GDWJ6Efns5p1Q3Y=', 'base64');
+	var BUFFER_SEED_SIZE_TWO = Buffer.from('mSYYt4k1R+iz53WadkM=', 'base64');
+	var BUFFER_SEED_MPINT = Buffer.from('AAAADiaZtxg1iehH57OadUN2', 'base64');
+
+	describe('toBuffer', function () {
+
+		var testbignum;
+
+		function standardExpect (buffer, shouldEqual) {
+			expect(buffer).to.be.instanceof(Buffer);
+			expect(buffer.equals(shouldEqual)).to.be.true;
```

Comment:
Since you know `shouldEqual` will always be a `Buffer`, you can do `expect(shouldEqual.equals(buffer)).to.be.true` and get rid of the first expectation (since an error will be thrown if it's not a `Buffer` anyway. And then I guess you don't need the helper function at all...